core/src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -36,38 +35,22 @@
 import org.elasticsearch.rest.RestStatus;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
 
-import static org.elasticsearch.action.admin.cluster.health.ClusterIndexHealth.readClusterIndexHealth;
-
 /**
  *
  */
-public class ClusterHealthResponse extends ActionResponse implements Iterable<ClusterIndexHealth>, StatusToXContent {
-
+public class ClusterHealthResponse extends ActionResponse implements StatusToXContent {
     private String clusterName;
-    int numberOfNodes = 0;
-    int numberOfDataNodes = 0;
-    int activeShards = 0;
-    int relocatingShards = 0;
-    int activePrimaryShards = 0;
-    int initializingShards = 0;
-    int unassignedShards = 0;
-    int numberOfPendingTasks = 0;
-    int numberOfInFlightFetch = 0;
-    int delayedUnassignedShards = 0;
-    TimeValue taskMaxWaitingTime = TimeValue.timeValueMillis(0);
-    double activeShardsPercent = 100;
-    boolean timedOut = false;
-    ClusterHealthStatus status = ClusterHealthStatus.RED;
-    private List<String> validationFailures;
-    Map<String, ClusterIndexHealth> indices = new HashMap<>();
+    private int numberOfPendingTasks = 0;
```

Comment:
I think that's because they don't relate to the cluster state , but then maybe we should call ClusterHealth, ClusterStateHealth?